plugins/lookup/passwordstore.py

```diff
@@ -380,6 +420,7 @@ def opt_lock(self, type):
             yield
 
     def setup(self, variables):
+        self.pass_cmd = self.get_option('backend')  # pass and gopass are commands as well
```

Comment:
How about:
```suggestion
        self.backend = self.get_option('backend')
        self.pass_cmd = self.backend  # pass and gopass are commands as well
```
Then you can use `self.backend` instead of `self.get_option('backend')` in the above places.